app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb

```diff
@@ -275,5 +275,22 @@
   </div>
 <% end %>
 
+<% if !@is_enhanced_ipp %>
```

Comment:
The comment I left on L232 of `app/views/idv/in_person/ready_to_verify/show.html.erb` is relevant here, too.